<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoanCalc</class>
 <widget class="QDialog" name="LoanCalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>603</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Loan Calculator</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>total credit amount</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="totalAmount"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>term</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="term"/>
        </item>
        <item>
         <widget class="QRadioButton" name="years">
          <property name="text">
           <string>years</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="months">
          <property name="text">
           <string>months</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>interest rate</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="rate"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>type (annuity, differentiated)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Annuity">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 183, 176);
color: rgb(45, 45, 45);
border-radius: 15px;
font-weight: bold;</string>
          </property>
          <property name="text">
           <string>Annuity</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Differentiated">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 183, 176);
color: rgb(45, 45, 45);
border-radius: 15px;
font-weight: bold;</string>
          </property>
          <property name="text">
           <string>Differential</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>monthly payment</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="monthlyPayment">
          <property name="text">
           <string>00.00</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>overpayment on credit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="Overpayment">
          <property name="text">
           <string>00.00</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>total payment</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="totalPayment">
          <property name="text">
           <string>00.00</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
